docs/public-networks/concepts/transactions/types.md

```diff
@@ -39,3 +39,35 @@ Transactions with type `EIP1559` are transactions introduced in [EIP-1559](https
 An `EIP1559` transaction always pays the base fee of the block it's included in, and it pays a priority fee as priced by `maxPriorityFeePerGas` or, if the base fee per gas + `maxPriorityFeePerGas` exceeds `maxFeePerGas`, it pays a priority fee as priced by `maxFeePerGas` minus the base fee per gas. The base fee is burned, and the priority fee is paid to the miner that included the transaction. A transaction's priority fee per gas incentivizes miners to include the transaction over other transactions with lower priority fees per gas.
 
 `EIP1559` transactions must specify both `maxPriorityFeePerGas` and `maxFeePerGas`. They must not specify `gasPrice`.
+
+## `BLOB` transactions
+
+Shard blob transactions introduced in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) enables scaling the
+Ethereum network by allowing large amounts of data (blobs) to be included that cannot be directly accessed or
+processed by the Ethereum Virtual Machine (EVM).
+
+When a blob-carrying transaction is included in a block, the transaction doesn't contain the blob data itself,
+but rather a commitment to this data. This commitment can be verified by the EVM, ensuring the data's availability and
+integrity without the EVM needing to access the data directly.
+
+:::info
+
+A commitment refers to a cryptographic proof that serves as a secure and verifiable way to attest to the existence
```

Comment:
I think this can be rewritten with simpler language.

```suggestion

ChatGPT
A commitment is a type of cryptographic proof that securely and verifiably confirms the existence and integrity of large data blobs.
```